### PR TITLE
Review

### PR DESCRIPTION
**Pull request by dooya4**

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+
```
Comment (doyamd) [inline popup.js:2]: Add comments to explain what the file does.

File: popup.js [hunk 2]
```diff
@@ -16,7 +17,9 @@ document.querySelector(".introduce").onclick = () => {
   slidesDiv.classList.remove("hidden");
   index = 0;
   show(index);
+
```
Comment (doyamd) [inline popup.js:20]: 'console.log' not necessary, could be removed to make it more clean.

File: popup.js [hunk 3]
```diff
@@ -44,6 +49,7 @@ nexts.forEach((next) => {
     show(index);
   };
 });
+
```
Comment (doyamd) [inline popup.js:52]: the variable name 'previouses' could be substituted to previousButtons. Make it more descriptive.

File: popup.js [hunk 4]
```diff
@@ -52,7 +58,9 @@ previouses.forEach((previous) => {
 });
 
 async function setShortcuts() {
+
```
Comment (doyamd) [inline popup.js:61]: There could be an error handling mechanism to handle if the 'getAll()' command falis.

**Review by dooya4 — COMMENTED** [review]
Overall it is a neat code, with good and descriptive variable naming.
And does the intended function.

**Pull request closed without merging**